draft-ietf-secevent-http-push.txt

```diff
@@ -34,14 +34,14 @@ Status of This Memo
    Internet-Drafts are working documents of the Internet Engineering
    Task Force (IETF).  Note that other groups may also distribute
    working documents as Internet-Drafts.  The list of current Internet-
-   Drafts is at https://datatracker.ietf.org/drafts/current/.
+   Drafts is at http://datatracker.ietf.org/drafts/current/.
```

Comment:
Why switching to http? The http url redirects to the https one.

Reply:
I was running an older version of `xml2rfc` than what Mike used to generate the current version. I've updated the PR with text and html versions generated from `xml2rfc` version 2.15.3.